### ISSUE TITLE
Reverse for 'auth_password_change' not found. 'auth_password_change' is not a valid view function or pattern name.

### ISSUE THREAD

**Issue by JayTurnr** [django-helpdesk/django-helpdesk#734]
`NoReverseMatch at /help/dashboard/`
`Reverse for 'auth_password_change' not found. 'auth_password_change' is not a valid view function or pattern name.`

```
{% if helpdesk_settings.HELPDESK_SHOW_CHANGE_PASSWORD and user.has_usable_password %}
<li><a href="{% url 'auth_password_change' %}"><i class="fa fa-user-secret fa-fw"></i> {% trans "Change password" %}</a></li>
{% endif %}
```

I think this might need to be `auth:password_change` but I'm not sure

I'm using `django-helpdesk==0.2.14`

**Comment by gwasser** [comment]
Thanks! Not sure how this was missed, I assume at some point broken when we moved to the 0.2 release. Will be fixed in next release (0.2.15) and merged into develop shortly.

**Comment by gwasser** [comment]
I think the old documentation had assumed `django-helpdesk` was running as part of a bigger project that already handled those URLs.

**Comment by gwasser** [comment]
Fixed in 0.2.15

**Comment by JayTurnr** [comment]
> I think the old documentation had assumed `django-helpdesk` was running as part of a bigger project that already handled those URLs.

In my case, I'm using AllAuth for that stuff.
